src/ghdcbot/engine/orchestrator.py

```
@@ -0,0 +1,535 @@
+
+
+def _send_notifications_for_new_events(
+    contributions: list[ContributionEvent],
+    storage: Storage,
+    discord_writer: DiscordWriter,
+    policy: MutationPolicy,
+    config: Any,
+    github_org: str,
+) -> None:
+    """Send Discord notifications for notification-worthy events (verified users only)."""
+    from ghdcbot.engine.notifications import send_notification_for_event
```

Ruff (0.15.5)

[error] Redefinition of unused `send_notification_for_event` from line 20: `send_notification_for_event` redefined here

Remove definition: `send_notification_for_event`

(F811)